app/components/channel_drawer/channels_list/channels_list.js

```diff
@@ -106,7 +106,11 @@ class ChannelsList extends React.PureComponent {
             backgroundColor: changeOpacity(theme.sidebarHeaderTextColor, 0.2),
             color: theme.sidebarHeaderTextColor,
             fontSize: 15,
-            lineHeight: 66
+            ...Platform.select({
+                android: {
+                    marginBottom: -5
+                }
+            })
         };
```

Comment:
I wonder if `searchBarInput` should be part of returned object of `getStyleSheet`. Same case with other changes below.

Reply:
No it should not, if we set it as part of the style then we cannot retrieve properties from it

Reply:
Got it, thanks!

Reply:
We could retrieve properties from it using `StyleSheet.flatten`, but I don't think we need it here